pages.php

```diff
@@ -32,6 +32,55 @@
     <head>
     </head>
     <body>
-        <?php /* Use PHP's include() method to render the correct page content in here */ ?>
+        <a href="/pages.php?page=home"> ++Home++ </a>
+        <a href="/pages.php?page=about"> ++About++ </a>
+        <a href="/pages.php?page=interests"> ++Interests++ </a>
+        
+        <?php /* Use PHP's include() method to render the correct page content in here */ 
+//
```

Comment:
You don't need to comment out blank lines @kko31623, you can just leave them blank :smile: